erpnext/accounts/party.py

```diff
@@ -174,13 +174,13 @@ def get_party_account(party_type, party, company):
 		account = frappe.db.get_value("Party Account",
 			{"parenttype": party_type, "parent": party, "company": company}, "account")
 
-		if not account:
+		if not account and party_type != 'Employee':
```

Comment:
Use explicit condition `if not account and party_type in ('Customer', 'Supplier')`